packages/market-service/src/exchange-rates-host/exchange-rates-host.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  FiatMarketDataArgs,
+  FiatPriceHistoryArgs,
+  HistoryData,
+  HistoryTimeframe,
+  MarketData
+} from '@shapeshiftoss/types'
+import dayjs from 'dayjs'
+
+import { RATE_LIMIT_THRESHOLDS_PER_MINUTE } from '../config'
+import { bnOrZero } from '../utils/bignumber'
+import { isValidDate } from '../utils/isValidDate'
+import { rateLimitedAxios } from '../utils/rateLimiters'
+import { ExchangeRateHostHistoryData, ExchangeRateHostRate } from './exchange-rates-host-types'
+
+const axios = rateLimitedAxios(RATE_LIMIT_THRESHOLDS_PER_MINUTE.DEFAULT)
+const baseCurrency = 'USD'
+const baseUrl = 'https://api.exchangerate.host'
+
+export const findByFiatSymbol = async ({
+  symbol
+}: FiatMarketDataArgs): Promise<MarketData | null> => {
+  try {
+    const { data } = await axios.get<ExchangeRateHostRate>(
+      `${baseUrl}/latest?base=${baseCurrency}&symbols=${symbol}`
+    )
+    const result = data as ExchangeRateHostRate
+    // we only need the price key in the `web`
+    return {
+      price: result.rates[symbol].toString(),
+      marketCap: '0',
+      changePercent24Hr: 0,
+      volume: '0'
+    }
+  } catch (e) {
+    console.warn(e)
+    throw new Error('FiatMarketService(findByFiatSymbol): error fetching market data')
+  }
+}
+
+export const findPriceHistoryByFiatSymbol = async ({
+  symbol,
+  timeframe
+}: FiatPriceHistoryArgs): Promise<HistoryData[]> => {
+  const end = dayjs().startOf('day')
```

Comment:
`By default, Day.js parses and displays in local time.` - can you confirm that the UTC timestamp we pass to`exchangerate.host` will return correct data on different browser/system times? Got burned with UTC offsets in the past so just double checking

Reply:
this is using the local time. dayjs supports timezones with a plugin though, should that be installed and integrated?

Reply:
IMO though local time is the way to go, because we're using these information for the client.

Reply:
well, actually it may return wrong information because it's using the YYYY-MM-DD format instead of a timestamp...

Reply:
there's like 0.1% difference between `exchangerate.host` and [Bitterex](https://www.tradingview.com/chart/?symbol=BITTREX%3AUSDEUR) candles daily close results.